src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/controller/SearchController.kt

```diff
@@ -24,6 +25,7 @@ class SearchController(
       ApiResponse(responseCode = "200", description = "OK"),
     ],
   )
+  @PreAuthorize("hasRole('ROLE_MANAGE_A_WORKFORCE_ALLOCATE') or hasRole('ROLE_WORKLOAD_READ')")
```

Comment:
This could also be done with 
`@PreAuthorize("hasAnyRole('ROLE_MANAGE_A_WORKFORCE_ALLOCATE', 'ROLE_WORKLOAD_READ')")`

Reply:
I personally it reads better as it is but, if we decide to add roles then thebother way is easier. 
I'll change it